docs/schema/commanduihandler-element.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: CommandUIHandler element
+description: Describes the CommandUIHandler element, which defines the handler for a command.
```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.